katacomb/katacomb/uv_facade.py

```diff
@@ -401,21 +401,21 @@ def Desc(self):
     def List(self):
         return self.uv.List
 
-    @property
     def nvis_from_NX(self):
-        """ Derive the number of vis from the attached NX table
-
-            NOTE: Sometimes in certain corner cases the 'nvis'
-            item in self.Desc.Dict incorrectly reports the number
-            of vis in the object (e.g. when the output from UVBlAvg
-            is empty, that happens when all the input data is flagged.).
-            Using the NX table to get the number of visibilities
-            is a workaround for this problem- it is derived
-            by counting the number of rows in the associated UV table.
+        """Derive the number of vis from the attached NX table
+
+        NOTE: Sometimes in certain corner cases the 'nvis'
+        item in self.Desc.Dict incorrectly reports the number
+        of vis in the object (e.g. when the output from UVBlAvg
+        is empty, that happens when all the input data is flagged.).
+        Using the NX table to get the number of visibilities
+        is a workaround for this problem, it is derived
+        by counting the number of rows in the associated UV table.
```

Comment:
Urgh, now it's a run-on sentence, which is worse :-)

Reply:
OK, OK!

I've rewritten the docstring with some more care.